### PR TITLE
fix: add precinct id to ballot filename

### PR DESCRIPTION
**Pull request by eventualbuddha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This app is intended to be used on an offline computer by an election admin.
    - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
    - approved official ballots in pdf format:
      - one for each combination of ballot style and precinct
-     - file name pattern: `election-${electionHash}-ballot-style-${ballotStyle}-precinct-id-${precinctId}.pdf`
+     - file name pattern: `election-${electionHash}-ballot-style-${ballotStyleId}-precinct-${dashify(precinctName)}-${precinctId}.pdf`
```
Comment (benadida) [inline README.md:24]: Precinct name?

Comment (eventualbuddha) [reply README.md:24]: I don't follow.

Comment (beausmith) [inline README.md:24]: Do we need to prefix each variable with a key/label?

Comment (beausmith) [reply README.md:24]: Prefix of "id" was removed. 
--

Bigger question: are filenames for humans or computers, or both? 

My vote is for both. Short suggestion:

election-${electionId}-precinct-${dashify(precinct Name)}-id-${precinctId}-style-${ballotStyle}.pdf

Comment (eventualbuddha) [reply README.md:24]: 🤷 it does help when the values are gibberish like `77`.

Comment (eventualbuddha) [reply README.md]: Alright, I changed it to what you suggested. Note that I interpreted `electionId` as a shorter version of `electionHash`.

Comment (beausmith) [reply README.md]: Sweet & sweet.

**Comment by beausmith** [comment]
D'oh! @benadida felt more comfortable with the first 10 characters.

**Comment by eventualbuddha** [comment]
> D'oh! @benadida felt more comfortable with the first 10 characters.

https://github.com/votingworks/election-manager/pull/19

**Pull request merged**